manifests/params.pp

```diff
@@ -16,68 +16,47 @@
       $systemd            = false # by default
       $pam_config         = 'none'
     } # end osfamily Gentoo
+
     'Debian': {
-      # Debian 7 uses version 0.6.8
-      # Debian 8 uses version 0.6.8
       # Debian 9 uses version 0.6.9
       # Debian 10 uses version 0.6.9
       # Debian 11 uses version 0.6.11
-      # Ubuntu 12.04 uses version 0.6.7
-      # Ubuntu 14.04 uses version 0.6.8
-      # Ubuntu 16.04 uses version 0.6.8
       # Ubuntu 18.04 uses version 0.6.9
       # Ubuntu 20.04 uses version 0.6.11
```

Comment:
```suggestion
      # Ubuntu 20.04 uses version 0.6.11
      # Ubuntu 22.04 uses version 0.6.11
```

see https://packages.ubuntu.com/jammy/libpam-pkcs11

As it works already on 22.04 maybe add this here and add the other checks for 22.04. But of course maybe better in a follow-up PR.

Reply:
I think the automatic tests won't work yet on Ubuntu 22.04, see #6: There are some packages missing for that.

But we can of course already add the line here.